jni/include/org_opensearch_knn_jni_FaissService.h

```diff
@@ -48,15 +48,15 @@ JNIEXPORT jlong JNICALL Java_org_opensearch_knn_jni_FaissService_loadIndex
  * Signature: (J[FI)[Lorg/opensearch/knn/index/query/KNNQueryResult;
  */
 JNIEXPORT jobjectArray JNICALL Java_org_opensearch_knn_jni_FaissService_queryIndex
-  (JNIEnv *, jclass, jlong, jfloatArray, jint);
```

Comment:
If we change this, we will have to change FaissService natives as well.

Reply:
Yes. Missed that by separating PR. I think that is why current build is failing. Let me add changes in java side as well.